makefiles/pojavlauncher.mk

```diff
@@ -26,29 +28,33 @@ pojavlauncher:
 	@echo "Using previously built pojavlauncher."
 else
 pojavlauncher: pojavlauncher-setup
-	# Reimplement the build script
+	# Copy and paste from the new Makefile... basically
 	cd $(BUILD_WORK)/pojavlauncher; \
 		unset CC CXX LD CFLAGS CPPFLAGS CXXFLAGS LDFLAGS; \
 		cd Natives; \
 			mkdir -p build; \
 			cd build; \
-			wget https://github.com/leetal/ios-cmake/raw/master/ios.toolchain.cmake; \
-			cmake .. -G Xcode -DDEPLOYMENT_TARGET=$(IPHONEOS_DEPLOYMENT_TARGET) -DCMAKE_TOOLCHAIN_FILE=ios.toolchain.cmake -DPLATFORM=OS64 -DENABLE_BITCODE=FALSE -DCMAKE_XCODE_ATTRIBUTE_CODE_SIGNING_REQUIRED="NO" -DCMAKE_XCODE_ATTRIBUTE_CODE_SIGNING_ALLOWED=NO -DCMAKE_XCODE_ATTRIBUTE_CODE_SIGN_IDENTITY=""; \
+			cmake . -DCMAKE_BUILD_TYPE=Release -DCMAKE_CROSSCOMPILING=true -DCMAKE_SYSTEM_NAME=Darwin -DCMAKE_SYSTEM_PROCESSOR=aarch64 -DCMAKE_OSX_SYSROOT="$(SDKPATH)" -DCMAKE_OSX_ARCHITECTURES=arm64 -DCMAKE_C_FLAGS="-arch arm64 -miphoneos-version-min=$(IPHONEOS_DEPLOYMENT_TARGET)" ..
 			cmake --build . --config Release --target pojavexec PojavLauncher; \
 			cd ../..; \
-			mkdir -p Natives/build/Release-iphoneos/PojavLauncher.app/Base.lproj; \
-			ibtool --compile Natives/build/Release-iphoneos/PojavLauncher.app/Base.lproj/MinecraftSurface.storyboardc Natives/en.lproj/MinecraftSurface.storyboard; \
-			mkdir -p Natives/build/Release-iphoneos/PojavLauncher.app/Frameworks; \
-			cp Natives/build/Release-iphoneos/libpojavexec.dylib Natives/build/Release-iphoneos/PojavLauncher.app/Frameworks/; \
-			cp -R Natives/resources/* Natives/build/Release-iphoneos/PojavLauncher.app/; \
+			mkdir -p Natives/build/PojavLauncher.app/Base.lproj; \
+			actool Natives/Assets.xcassets --compile Natives/resources --platform iphoneos --minimum-deployment-target $(IPHONEOS_DEPLOYMENT_TARGET) --app-icon AppIcon --output-partial-info-plist /dev/null; \
```

Comment:
There is a platform var you should use here

Reply:
done